client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -5,83 +5,119 @@ import Filters from '../../components/Filter/Filter';
 import NavBar from '../../components/Navbar/Navbar';
 import React from 'react';
 import ProfilePage from '../ProfilePage/ProfilePage';
+import Map from 'pigeon-maps';
+import Marker from 'pigeon-marker';
 
 export default function FiltersPage() {
-  const [listOrMap, setListOrMap] = React.useState(true);
+  const [listMode, setListMode] = React.useState(true);
+
+  function mapTilerProvider(x, y, z, dpr) {
+    return `https://c.tile.openstreetmap.org/${z}/${x}/${y}.png`;
+  }
+
+  const markers = [
+    [50.874, 4.6947],
+    [50.212, 4],
+    [51, 5],
+  ];
+
+  function markerClick({ anchor }) {
+    console.log(anchor);
+  }
 
   function renderView() {
-    if (listOrMap == true) {
+    if (listMode == true) {
```

Comment:
see my comment on `renderListView` and `renderMapView` below. then later on line 178 you can do
```javascript
{listMode ? renderListView() : renderMapView()}
```